app/budget/budget.css

```diff
@@ -0,0 +1,374 @@
+}
+
+.budget-summary .budget-summary-title button {
+    margin-left: 10px;
+}
+
+.budget-summary .table > tbody > tr > td, .table > tfoot > tr > td {
+    border-top: unset;
+}
+
+.budget-summary .total-balance thead{
+    border-top: 1px solid #E5E5E5 !important;
+    border-bottom: unset;
+}
+
+.flex-row {
+    display: flex;
+    flex-direction: row;
+}
+
+.bg-gray {
+    background-color: #f9f9f9;
+}
+
+.description {
+    padding: 0px !important;
+}
+
+.form-group span {
+    display: block;
+    max-width: 70%;
+    overflow-x: hidden;
+    text-decoration: underline;
+}
+
+.form-group .span-full-width {
+    display: block;
+    max-width: 100%;
+}
+
+.form-group textarea {
+    max-width: 100%;
+}
+
+.testAb span {
+    text-decoration: underline;
+}
+
+[contenteditable]:focus {
+    outline: 1px solid #E5E5E5 !important;
+    background-color: #F9F9F9;
+}
+
+[contenteditable] {
+    border: 1px solid white;
+    text-decoration: underline;
+}
+
+[contenteditable]:hover {
+    outline: 1px solid #E5E5E5 !important;
+    background-color: #F9F9F9;
+}
+
+.description .line-item-container:first-child {
+    padding-top: 20px !important;
+}
+
+.line-item-container {
+    padding: 10px 20px 10px 55px;
+}
+
+
+/* budget summary */
+.budget-pie-chart {
+    height: 250px;
+    align-self: center;
+    justify-content: center;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.budget-summary table {
+    font-size: 12px;
+}
+
+.budget-summary table tr th:nth-child(2) {
+    width:5%;
+
+}
+
+.budget-table {
+    margin-bottom: 0px !important;
+    border-bottom: unset;
+}
+
+.budget-table thead {
+    border-bottom: 3px solid #ebebeb;
+}
+
+.budget-table thead tr th{
+    background: white !important;
+}
+
+.budget-table thead tr th:first-child{
+    width: 3%;
+}
+
+.budget-table thead tr th:nth-child(3){
+    width: 60%;
+}
+
+.budget-table thead tr th:last-child{
+    width: 10%;
+}
+
+.budget-table > thead > tr > th, .table > tbody > tr > th, .budget-table > tfoot > tr > th, .budget-table > thead > tr > td, .budget-table > tbody > tr > td, .budget-table > tfoot > tr > td {
+    padding: 12px;
+    line-height: 2;
+}
+
+/* cash out - instructional */
+
+.cashout-instructional .course-title {
+    display: inline-block;
+    font-weight: bold;
+    margin: 0;
+    line-height: unset;
+}
+
+.cashout-instructional thead tr th {
+    font-weight: bold;
+    padding-bottom: 20px !important;
+    text-align: center;
+}
+
+.cashout-instructional tbody tr td{
+    border-bottom: 15px solid white;
+    border-top: unset !important;
+    text-align: center;
+    vertical-align: middle;
+ }
+
+.cashout-instructional tbody tr td:nth-child(7) {
+    background-color: white !important;
+}
+
+
+.cashout-instructional tbody tr td:nth-child(1), .cashout-instructional td:nth-child(2), .cashout-instructional td:nth-child(3), .cashout-instructional td:nth-child(4), .cashout-instructional td:nth-child(5), .cashout-instructional td:nth-child(6) {
```

Comment:
This looks like a hardcoded solution that wouldn't work with variable number of rows? Or am I miss-understanding?

Instead, could you make classes with the appropriate styling, use them specifically in the mock, and then I can figure out how to dynamically apply those classes appropriately with angular when I use real data?